godel.rosinstall

```diff
@@ -6,3 +6,5 @@
     version: indigo-devel}
 - git: {local-name: motoman, uri: 'https://github.com/ros-industrial/motoman.git',
     version: indigo-devel}
+- git: {local-name: abb, uri: 'https://github.com/ros-industrial/abb.git',
+    version: indigo-devel}
```

Comment:
Nope. I missed the `abb` dependency in the  `godel_irb2400_support` package. I'll fix it later.